CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 2.2.0
+
+- A lot of internal changes:
+  - iOS now downloads the dependency from CocoaPods
+  - Android downloads the common code from Maven
+  - Removed a bunch of scripts
+  - Removed the Android project and added the files to the plugin folder
+  - Added platform version and name to the http headers
+  - Added a script to create the unitypackage
+  - Moved inner classes to their own files
+  - Use SimpleJSON instead of JSONUtility
+- Adds proxyURL property, useful for kids category apps, so that they can set up a proxy to send requests through. Do not use this unless you've talked to RevenueCat support about it.
+- Adds userDefaultsSuiteName. Set this to use a specific NSUserDefaults suite for RevenueCat. This might be handy if you are deleting all NSUserDefaults in your app and leaving RevenueCat in a bad state.
+- Update to ExternalDependencyManager 1.2.156
```

Comment:
let's maybe move these up to the top, since they're the most likely to be interesting for devs